playground/frontend/lib/modules/editor/components/editor_textarea.dart

```diff
@@ -51,6 +60,7 @@ class EditorTextArea extends StatefulWidget {
 
 class _EditorTextAreaState extends State<EditorTextArea> {
   CodeController? _codeController;
+  FocusNode focusNode = FocusNode();
```

Comment:
@miamihotline not critical but in dart we do not need to repeat the class name.  This works as well and is probably more clear: `var focusNode = FocusNode()`

Reply:
Okay, thank you for the suggestion, I'll change this line